ctru-rs/src/gfx.rs

```diff
@@ -31,6 +64,15 @@ pub enum Side {
     Right,
 }
 
+/// A handle to libctru's gfx module. This module is a wrapper around the GSPGPU service that
+/// provides helper functions and utilities for software rendering.
+///
+/// The service exits when this struct is dropped.
+pub struct Gfx {
+    pub top_screen: RefCell<TopScreen>,
+    pub bottom_screen: RefCell<BottomScreen>,
+}
```

Comment:
This needs a private field, otherwise users can just directly create the struct (maybe causing bad things on drop since gfx wasn't properly initialized):
```suggestion
pub struct Gfx {
    pub top_screen: RefCell<TopScreen>,
    pub bottom_screen: RefCell<BottomScreen>,
    _private: ()
}
```